Library/ViewModels/PledgeDisclaimerViewModelTests.swift

```diff
@@ -9,6 +9,7 @@ final class PledgeDisclaimerViewModelTests: TestCase {
   private let notifyDelegateLinkTappedWithURL = TestObserver<URL, Never>()
 
   override func setUp() {
+    super.setUp()
```

Comment:
`PledgeDisclaimerViewModelTests` extends `TestCase`, which sets up a default environment. Because this test wasn't calling `super.setUp()` but is (implicitly) calling `super.tearDown()`, it was unintentionally popping an extra environment off the stack.